### ISSUE TITLE
spring() and demos break on main NPM library

### ISSUE THREAD

**Issue by jasonpang** [chenglou/react-motion#266]
I installed `react-motion` via `npm install react-motion` to a test project. I copied your `demo8-draggable-list` to my test project, but there were no animations when trying to move elements. The elements would re-order instantly, and there were also no shadow and scaling animations for the demo.

When I built the pre-release version and replaced the npm-installed-version of `react-motion` with the pre-release version built from this master branch, then the demo worked perfectly with proper animations.

Could you update the main version on NPM to work with the demos?

The following code works differently between the main-npm-version and the pre-release version:

https://github.com/chenglou/react-motion/blob/master/demos/demo8-draggable-list/Demo.jsx#L76 (edited below)
```javascript
const style = lastPressed === i && isPressed
            ? {
                shadow: spring(15, springConfig),
              }
            : {
                shadow: spring(1, springConfig),
              };
          return (
          console.log(shadow);
            <Motion style={style} key={i}>
              {({shadow}) =>
```

In the pre-release version (where things are working fine):
- `console.log(shadow)` resolves to `{stiffness: 1.0, damping: 1.0, onRest: null, precision: 1.0, val: 1.0}`
- Inside the `<Motion>` tag, `{shadow}` gives a numerical value -- exactly what we expect and need for the demo to work

In the main-npm-version (this is what probably breaks the code):
- `console.log(shadow)` resolves to `{val: 1.0, config: {stiffness: 1.0, damping: 1.0}}`
- Inside the `<Motion>` tag, `{shadow}` is invalid: it is `NaN` of type `number`

^ I edited the `spring.js` to make `spring()` return a flat object like in the pre-release version (i.e. I edited the code to make `spring()` *not* have a `config` property, and just have `stiffness` and `damping` and `onRest` on the main object itself and not nested), and it made the `NaN` error go away.

Hopefully this sheds some light on the issue? Thanks for the awesome library!

**Comment by chenglou** [comment]
I'm iterating on the master repo and there are some breaking changes to the API. The README directs to the latest stable build's url. That one works.

**Comment by notsoluckycharm** [comment]
Were you able to get this to work, @jasonpang ? I have git@github.com:chenglou/react-motion.git#cc852fe787bb15a8b4a9e51538e03c57d5543d2e in my package.json but now I am receiving

Module not found: Error: Cannot resolve module 'react-motion' in /Users/bradgunn/code/wellpath_web/src/components
 @ ./src/components/Demo.jsx 13:19-42

As an aside though, Cheng, thank you for your library - however I do have to criticize the fact that bleeding should be a commit/branch and stable should be master. Otherwise, just trying to get these amazing demos to work.

**Comment by chenglou** [comment]
@jasonpang @notsoluckycharm sorry for all the trouble. This has been been fixed for React-motion v0.4.1. My initial thought was to change the default displayed branch on github to point to the stable commit, and keep master bleeding edge. Would you prefer the other way around?

(Closing this issue as it's resolved. Please do continue the discussion).
